frame/stake-tracker/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "pallet-stake-tracker"
+version = "4.0.0-dev"
```

Comment:
```suggestion
version = "1.0.0-dev"
```

Reply:
Again, 4.0.0-dev is more consistent with versioning of all the pallets and modules across the board.

Reply:
Idk, I think starting from 1.0.0 is more sensible.

Reply:
yeah, but not consistent with the rest of the pallets/modules though. I don't have a strong opinion, but as a builder I would prefer consistency as it's less confusing.